docs/servo_power_test/src/dut_controller.ts

```diff
@@ -28,6 +28,33 @@ export class DutController {
     const chunk = await this.dut.read();
     return chunk;
   }
+  private async checkDutBuffer() {
```

Comment:
the name "check" is not friendly to developers to know what value will be returned in which case.
How about `readDataWithTimeout()` and add a function-level documentation that describes the meaning of the return value?

Reply:
Also, in that case, it may be a good idea to make the timeout configurable, since the name sounds like requiring the timeout given as a parameter.